torch_tvm/operators.cpp

```diff
@@ -183,27 +217,60 @@ RegisterTVMOperator reg({
        conv_attrs->groups = relayToConstant<int>(inputs[8]);
        conv_attrs->data_layout = "NCHW";
        conv_attrs->kernel_layout = "OIHW";
```

Comment:
This seems to be invalid for 1d conv, right?

Reply:
@llyfacebook, we are converting it to 2D conv so it is still valid.

Reply:
ok, I assume you could still distinguish it from conv2d in tvm side, since you will need to specialize the perf optimization for conv1d.

Reply:
@llyfacebook, tvm has no conv1d op. However neither does mkldnn as far as I understand. We will have to just benchmark and see how they perform.